lib/mayaUsd/undo/UsdUndoableItem.h

```diff
@@ -14,16 +14,14 @@
 // limitations under the License.
 //
 
-#ifndef MAYAUSD_UNDO_UNDOABLE_BLOCK_H
```

Comment:
Fixed copy-paste damage...  Was causing really, really weird compilation errors for my local changes...